README.md

```diff
@@ -180,44 +180,31 @@ For more information on how it works and how you can use this in your own applic
 
 
 ## PROGMEM with flash sizes greater than 64kB
-The usual `PROGMEM` attribute stores constant data such as string arrays to flash and is great if you want to preserve the precious RAM. However, PROGMEM will only store content in the lower section, from 0 and up to 64kB. If you want to store data in other sections, you can use `PROGMEM1` (64 - 128kB), `PROGMEM2` (128 - 192kB), or `PROGMEM3` (192 - 256kB), depending on the chip you're using.
-Accessing this data is not as straightforward as with `PROGMEM`, but it's still doable:
+The  `PROGMEM` attribute stores constant data, such as string arrays, in flash and is great if you want to preserve the precious RAM. However, the usual PROGMEM library macros can access data only in the lower part of flash memory up to 64kiB.  If you store more data in the PROGMEM section, you will get a warning during the compilation process because some of the PROGMEM data will not be accessible in the usual way.
+
+If you want to store and access data independently from the 64kiB limit, use `PROGMEM_FAR`  from the library `program_far.h` (installable using the Arduino library manager). Data with that attribute is stored at the far end of flash memory. Accessing this data is not as straightforward as with `PROGMEM`, but it's still doable:
```

Comment:
```suggestion
If you want to store and access data independently from the 64kiB limit, use `PROGMEM_FAR` from the "progmem_far" library (installable using the Arduino library manager). Data with that attribute is stored at the far end of flash memory. Accessing this data is not as straightforward as with `PROGMEM`, but it's still doable:
```

Correct the library name. Libraries are identified by the name defined in the metadata:

https://github.com/felias-fogg/progmem_far/blob/main/library.properties#L1

This is what the user will see in Library Manager.